jetty-p2/pom.xml

```diff
@@ -0,0 +1,58 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <parent>
+    <groupId>org.eclipse.jetty</groupId>
+    <artifactId>jetty-project</artifactId>
+    <version>10.0.7-SNAPSHOT</version>
+  </parent>
+  <modelVersion>4.0.0</modelVersion>
+  <artifactId>jetty-p2</artifactId>
+  <name>Jetty :: P2</name>
+  <description>Generates a (maven based) P2 Updatesite</description>
+  <packaging>pom</packaging>
+  <properties>
+    <tycho-version>2.5.0-SNAPSHOT</tycho-version>
```

Comment:
there is a property `tycho.version` in parent pom.

Reply:
I'm just not sure if it is okay to increase the version there also and as we require a very specific version here I added an override here.

So do you think the version in the parent should simply be increased?

Reply:
You can still use the top level `${tycho.version}` in your build scripts with `mvn -Dtycho.version=blabla`.

Reply:
@joakime I wonder if we really the profile `eclipse-sign` in parent pom?
as long as those plugins still exist it's fine 
```
            <groupId>org.eclipse.tycho.extras</groupId>
            <artifactId>tycho-pack200a-plugin</artifactId>

            <groupId>org.eclipse.tycho.extras</groupId>
            <artifactId>tycho-pack200b-plugin</artifactId>
```

Reply:
for jetty 10.0.x? probably not.
for jetty 9.4.x? i think so?

Reply:
I thought eclipse was moving away from pack200 entirely.
But I cannot find any kind of official statement about this.

Reply:
https://www.eclipse.org/lists/cross-project-issues-dev/msg18273.html

Reply:
@akurtakov thanks.
I'll make a PR to remove - https://github.com/eclipse/jetty.project/issues/6853

Reply:
BTW I have created a discussion about the future of pack200 in tycho here:
- https://github.com/eclipse/tycho/discussions/272

Reply:
pack200 removal PRs

* #6854 - for Jetty 9.4.x
* #6855 - for Jetty 10.0.x+

Reply:
@laeubi we cannot release this with a SNAPSHOT version (that violates the maven central pom rules).

Do we want `jetty-p2` to not deploy to maven central?

Reply:
There will be a new tycho release hopeful soon and I'll the update this PR to use a release version!

Reply:
The new releae is there and I removed the SNAPSHOT identifier